File: app/src/main/java/io/github/maratsubkhankulov/miner/entities/Job.java
package io.github.maratsubkhankulov.miner.entities;

import io.github.maratsubkhankulov.miner.entities.BlockHeader;

/**
 * Created by msubkhankulov on 5/6/18.
 */

public class Job {
    public String jobId;
    public String clientId;
    public Integer nonceRange;
    public BlockHeader blockHeader;
}

File: app/src/main/java/io/github/maratsubkhankulov/miner/entities/Job.kt
package io.github.maratsubkhankulov.miner.entities

import io.github.maratsubkhankulov.miner.entities.BlockHeader

/**
 * Created by msubkhankulov on 5/6/18.
 */

class Job {
    var jobId: String? = null
    var clientId: String? = null
    var nonceRange: Int? = null
    var blockHeader: BlockHeader? = null
}
